src/ui/Loading/animations/FlashingThreeDots..tsx

```diff
@@ -0,0 +1,56 @@
+import { VFC } from 'react';
+import tw, { styled } from 'twin.macro';
+import { LoadingAnimationProps } from '..';
+
+const FlashingThreeDots: VFC<LoadingAnimationProps> = (props) => (
+  <FlashingDotsContainer {...props}>
+    {[...new Array(3)].map((val, idx) => (
+      <FlashingDot key={idx} index={idx} />
+    ))}
+  </FlashingDotsContainer>
+);
+
+export default FlashingThreeDots;
+
+const FlashingDotsContainer = styled.div<LoadingAnimationProps>`
+  ${tw`flex items-center justify-center`}
+  ${(p) => getGapBySize(p.size)}
+  & > span {
+    ${(p) => getBgColorByVariant(p.variant)}
+    ${(p) => getDimensionBySize(p.size)}
+  }
+`;
+
+type FlashingDotProps = {
+  index: number;
+};
+const FlashingDot = styled.span<FlashingDotProps>`
+  ${tw`rounded-full animate-pulse`}
+  animation-duration: 1250ms;
+  animation-delay: ${(p) => p.index * 150}ms;
+`;
+
+const getGapBySize = (size?: LoadingAnimationProps['size']) => {
+  if (size === 'sm') {
```

Comment:
These should be switch

Reply:
`switch`looks pretty bad with `return` in them, `if` is more elegant in this case

```js
  if (size === 'sm') {
    return tw`gap-1`;
  }
  if (size === 'lg') {
    return tw`gap-3`;
  }
  return tw`gap-2`;
```

vs 

```js
switch (size) {
    case 'sm':
        return tw`gap-1`;
    case 'lg':
        return tw`gap-3`;
    default:
        return tw`gap-2`;
}
```

Reply:
individual `if` doesn't hint that they are related to each other. 
```ts
if (a === 'a') return;
if (b === 'b') return;
return;
```

`switch` indicates really clear that we are switching on the same variable. Why is it bad though, syntax-wise they are pretty much the same?

Reply:
If the condition is based on two or more variables, then `if` guard is better.

Reply:
I think they're ok, it all comes down to coding style I guess. Less keyword means less brain cell needed for me.

And yea they're the same, any half-decent programmer would be about to tell what they do exactly. If they need some kind of hint, they probably need some more skills lol